curriculum/challenges/portuguese/01-responsive-web-design/basic-css/use-clockwise-notation-to-specify-the-padding-of-an-element.portuguese.md

```diff
@@ -85,7 +85,42 @@ tests:
 ## Solution
 <section id='solution'>
 
-```js
-// solution required
+```html
+<style>
+  .injected-text {
+    margin-bottom: -25px;
+    text-align: center;
+  }
+
+  .box {
+    border-style: solid;
+    border-color: black;
+    border-width: 5px;
+    text-align: center;
+  }
+
+  .yellow-box {
+    background-color: yellow;
+    padding: 20px 40px 20px 40px;
+  }
+
+  .red-box {
+    background-color: crimson;
+    color: #fff;
+    padding: 20px 40px 20px 40px;
+  }
+
+  .blue-box {
+    background-color: blue;
+    color: #fff;
+	padding: 40 20 20 40;
```

Comment:
@CubeSkyy Thanks for the PR!

This looks good to me, but just a nit-pick here to keep our code consistent:

```suggestion
    padding: 40px 20px 20px 40px;
```